code/modules/mob/living/simple_animal/hostile/mining/elites/pandora.dm

```diff
@@ -116,7 +116,7 @@
 		recalculation_speed = 2
 
 /mob/living/simple_animal/hostile/asteroid/elite/pandora/proc/chaser_burst(target)
-	ranged_cooldown = world.time + cooldown_time * 2
+	ranged_cooldown = world.time + cooldown_time * 2.5 // this fucks at low hp, so needs higher cooldown than other attacks
```

Comment:
comment nitpick, it would probably be a better idea describe what this does at low HP specifically as saying that "something fucks" can be construed pretty easily.

Reply:
made better.